core/src/main/java/com/datastax/oss/driver/internal/core/resolver/Response.java

```diff
@@ -0,0 +1,18 @@
+package com.datastax.oss.driver.internal.core.resolver;
```

Comment:
Please move everything that is related to  a MockResolver to a separate directory.